src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1742,7 +1746,11 @@ public String getQuestionText(String feedbackQuestionId) {
         return sortedMap;
     }
 
-    public LinkedHashMap<String, Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>>
+    /**
+     * Returns an ordered Map with {@code giverTeam} name as key
+     * sorted by {@code compareByTeamQuestionGiverTeamRecipient} comparator.
```

Comment:
Take a look here
``` java
         if (sortByTeam) {
             Collections.sort(responses, compareByTeamRecipientGiverQuestion);
         } else {
             Collections.sort(responses, compareByRecipientGiverQuestion);
         }
```

The comparator is `compareByTeamRecipientGiverQuestion ` or `compareByRecipientGiverQuestion `
Sorry I didn't make it clear in my previous review